sftp-connector/src/com/axonivy/connector/sftp/service/SftpClientService.java

```diff
@@ -18,18 +20,29 @@
 import com.jcraft.jsch.Session;
 import com.jcraft.jsch.SftpException;
 
+import ch.ivyteam.ivy.environment.Ivy;
 
 /**
- * Service class for file transfer to/from the SFTP server.
- * The service class is used to decouple the SFTP implementation.
+ * Service class for file transfer to/from the SFTP server. The service class is
+ * used to decouple the SFTP implementation.
  */
 public class SftpClientService implements AutoCloseable {
-	private static final Logger LOG = Logger.getLogger(SftpClientService.class);
+	private static final Logger LOG = Ivy.log();
+
 	private static final String PATHSEPARATOR = "/";
 	private static final int SESSION_TIMEOUT = 10000;
 	private static final int CHANNEL_TIMEOUT = 5000;
-	
-	
+
+	private static final String SFTP_VAR = "com.axonivy.connector.sftp.server";
+	private static final String HOST_VAR = "host";
+	private static final String PORT_VAR = "port";
+	private static final String SECRET_SSHPASSPHRASE_VAR = "secret.sshpassphrase";
+	private static final String SECRET_SSHKEY_VAR = "secret.sshkey";
+	private static final String AUTH_VAR = "auth";
+	private static final String PASSWORD_VAR = "password";
+	private static final String USERNAME_VAR = "username";
+	private static final String PASSWORD = "password";

```

Comment:
Can you rename PASSWORD to AUTH_METHOD? or better list the authentication methods in a proper Enum